### PR TITLE
Tutorials for unsteady FSI with Python wrapper and Nastran structure

### PR DESCRIPTION
**Pull request by Nicola-Fonzi**
PR related to pull request #1124 in the SU2 repo

### CODE REVIEW

**Review by pcarruscag — APPROVED** [review]
Very nice :+1: I'm having some issues with the preview, if you want we can merge directly to master to see it in the website.

File: _tutorials/multiphysics/Unsteady_FSI_Python/Dynamic_FSI_Python.md
Comment (Nicola-Fonzi) [reply _tutorials/multiphysics/Unsteady_FSI_Python/Dynamic_FSI_Python.md:109]: I actually saw them [here](https://raw.githubusercontent.com/su2code/su2code.github.io/master/_tutorials/multiphysics/steady_fsi/Static_FSI.md), which is the tutorial I mention in the .md file.

File: _tutorials/multiphysics/Unsteady_FSI_Python/Dynamic_FSI_Python.md
Comment (Nicola-Fonzi) [reply _tutorials/multiphysics/Unsteady_FSI_Python/Dynamic_FSI_Python.md:112]: I tried to already use the paths that the images should have once the merging process is completed. Maybe they are not displayed until this is done.

File: _tutorials/multiphysics/Unsteady_FSI_Python/Dynamic_FSI_Python.md
Comment (Nicola-Fonzi) [reply _tutorials/multiphysics/Unsteady_FSI_Python/Dynamic_FSI_Python.md:199]: Yes you are right, I will change that

**Comment by Nicola-Fonzi** [comment]
I can try to change the font of the keywords and then, if it is fine for you, merging in master would be perfect to see how it is displayed

**Comment by pcarruscag** [comment]
Go for it.

**Comment by Nicola-Fonzi** [comment]
Ok the fonts are changed. 

I noticed that, as I branched from the develop of your repo, there are two your commits that are not present in master. The very first two commits of this PR. Is it a problem?

**Comment by pcarruscag** [comment]
It should be fine

**Comment by Nicola-Fonzi** [comment]
Perfect, is there something needed from my side to start merging?

**Comment by pcarruscag** [comment]
Nop, if the merge button shows as green, it is good to go.
It may take a little bit for this to go live on the website.

**Comment by Nicola-Fonzi** [comment]
It is already online! Pictures appears as expected, I just did one latex statement incorrect, fixing it now

**Comment by TobiKattmann** [comment]
...I thought I had time until tomorrow to look over this :D but things move faster than old Tobi can process 👴 
Thanks for taking the time to write a tutorial! 

for completeness: You can host the website locally using jekyll to test everything before pushing

**Comment by Nicola-Fonzi** [comment]
Thanks for the tip about jekyll, I will have a look at that!

**Pull request merged**
